server.js

```diff
@@ -35,7 +35,7 @@ require("./app/routes/recipeIngredient.routes")(app);
 require("./app/routes/user.routes")(app);
 
 // set port, listen for requests
-const PORT = process.env.PORT || 3200;
+const PORT = process.env.PORT || 3201;
```

Comment:
Can we leave this at 3200? Just because the readme says that we need to config our ProxyPass to forward to localhost:3200. If this changes, everyone will have to change their apache configs and the apache configs on the live server will have to change too.

Reply:
If you check the file on blackboard with the instructions to install the recipe app, they asked us to change the route to 3201. Maybe its for consistency with all teams. Unless you already have something that is running on 3201 I think that wouldn't be a lot of work to run it there too. We'll get input from the rest of the team about this.